extra/2-piping.js

```diff
@@ -16,22 +16,23 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+ return a + b
 }
 
-function multiply() {
-
+function multiply(c, d) {
+ return c* d
```

Comment:
please correct the formatting of this to ensure that it's consistent. There must  be a space between ``c`` and ``*``